api.md

```diff
@@ -469,7 +535,66 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
 
+Detach a data node from a hypertable or from all hypertables.
+
+This function is typically used if a data node should no longer be
+used. Then it need to be removed from all hypertables that use
+it. Another reason for detaching a data node from a hyper table is
+that you want to have fewer
+
+#### Required Arguments [](detach_data_node-required-arguments)
+
+| Name              | Description                                   |
+| `node_name`       | Local name of data node to attach             |
+
+##### Parameter Description
+
+* `node_name` - Local name of the data node to attach to the
+  hypertable.
+
+#### Optional Arguments [](detach_data_node-optional-arguments)
+
+| Name         | Description                            |
+| `hypertable` | Name of hypertable to detach node from |
+| `force`      | Force remove the data node             |
+
+##### Parameter Description
+
+* `hypertable` - The name of the hypertable where the data node should
+  be attached. If NULL, the data node will be detached from all
+  hypertables.
+
+* `if_not_attached` - Do not generate an error if the data node is
+  already attached to the hypertable. A notice will be printed that
+  the data node is attached.
```

Comment:
I think a commit might not have been added cause I believe this should be `force`?

Reply:
Fixed.